### PR TITLE
[Added] About Section

### PR DESCRIPTION
**Pull request by amitsuthar69**
### Description
Added About Section as mentioned in #4 

### Checklist
- [x] Made array for data in `stats.js` and mapped it in `index.js`.
- [x] Used Grids to design boxes.
- [x] Used `@apply` directive to make gradient.
- [x] Used `hover:` pseudo class on boxes to get a hover effect of gradient.

### Features :

**Desktop View :**
image

**Tablet View :**
image

**Mobile View :**
image

image

### CODE REVIEW

File: src/components/AboutSection/index.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const items = [
+    {
+      title: "About Gravity Team",
+      paragraph:
+        "At Gravity Team, we are on the mission to balance the supply and demand across crypto markets worldwide. We are a crypto native market maker founded by traders, developers, and innovators who are strong believers and supporters of the future of decentralization and digital assets.",
+    },
+  ];
+
+  return (
+    <section className="bg-[#141619] text-white text-center">
```
Comment (kamalika0363) [inline src/components/AboutSection/index.jsx]: bg-[#141619] text-white text-center min-h-screen p-8 sm:p-16
```suggestion
    <section className="bg-[#141619] text-white text-center">
```

File: src/components/AboutSection/index.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            {item.title}
+          </h1>
+          <p className="mt-6 md:w-2/3 lg:w-2/4 m-auto">{item.paragraph}</p>
+        </div>
+      ))}
+      <div className="mt-8 mx-4 md:mx-5 lg:mx-32 grid grid-cols-2 md:grid-cols-4 gap-2">
+        {statsArray.map((item) => (
+          <div
+            key={item.id}
+            className="p-4 border border-gray-300 hover:primary-gradient rounded"
```
Comment (kamalika0363) [inline src/components/AboutSection/index.jsx]: className="p-4 border border-slate-700 hover:primary-gradient rounded
```suggestion
            className="p-4 border border-gray-300 hover:primary-gradient rounded"
```

File: src/components/AboutSection/index.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  ];
+
+  return (
+    <section className="bg-[#141619] text-white text-center">
+      {items.map((item, index) => (
+        <div key={index}>
+          <h1 className="font-inter text-3xl md:text-3xl lg:text-5xl font-medium">
+            {item.title}
+          </h1>
+          <p className="mt-6 md:w-2/3 lg:w-2/4 m-auto">{item.paragraph}</p>
```
Comment (kamalika0363) [inline src/components/AboutSection/index.jsx]: className="mt-6 md:w-2/3 lg:w-2/4 m-auto w-80 justify-evenly"
```suggestion
          <p className="mt-6 md:w-2/3 lg:w-2/4 m-auto">{item.paragraph}</p>
```

File: src/components/AboutSection/index.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        </div>
+      ))}
+      <div className="mt-8 mx-4 md:mx-5 lg:mx-32 grid grid-cols-2 md:grid-cols-4 gap-2">
+        {statsArray.map((item) => (
+          <div
+            key={item.id}
+            className="p-4 border border-gray-300 hover:primary-gradient rounded"
+          >
+            <h2 className="text-md md:text-xl font-semibold">{item.title}</h2>
```
Comment (kamalika0363) [inline src/components/AboutSection/index.jsx:28]: <h2 className="text-md md:text-lg font-semibold ">
```suggestion
            <h2 className="text-md md:text-xl font-semibold">{item.title}</h2>
```

Comment (kamalika0363) [inline src/components/AboutSection/index.jsx:28]: <h2 className="text-md md:text-lg font-semibold ">
```suggestion
            <h2 className="text-md md:text-xl font-semibold">{item.title}</h2>
```

Comment (kamalika0363) [inline src/components/AboutSection/index.jsx]: className="text-md md:text-lg font-semibold 
```suggestion
            <h2 className="text-md md:text-xl font-semibold">{item.title}</h2>
```
```suggestion
            <h2 className="text-md md:text-xl font-semibold">{item.title}</h2>
```

**Comment by kamalika0363** [comment]
@amitsuthar69 consider this changes

File: src/components/AboutSection/index.jsx (hunk 1 above)
Comment (amitsuthar69) [reply src/components/AboutSection/index.jsx]: **Mobile view if `h-screen` :**
image

---

**Desktop view if `h-full` :**
image

**Comment by amitsuthar69** [comment]
### Fixes :
**Mobile :**
image
image

---

**Desktop :**
image

**Comment by kamalika0363** [comment]
LGTM!!

**Pull request merged**